tests/Aspire.Hosting.Dapr.Tests/DaprSchemaTests.cs

```diff
@@ -0,0 +1,30 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Hosting.Tests.Schema;
+using Xunit;
+
+namespace Aspire.Hosting.Dapr.Tests;
+
+public class DaprSchemaTests
+{
+    [Fact]
+    public void ValidateWithDaprManifest()
+    {
+        new SchemaTests().ValidateApplicationSamples("DaprWithComponents", (IDistributedApplicationBuilder builder) =>
+            {
+                var dapr = builder.AddDapr(dopts =>
+                {
+                    // Just to avoid dynamic discovery which will throw.
+                    dopts.DaprPath = "notrealpath";
+                });
+                var state = dapr.AddDaprStateStore("daprstate");
+                var pubsub = dapr.AddDaprPubSub("daprpubsub");
+
+                builder.AddProject<Projects.ServiceA>("project")
```

Comment:
Use a mock Project like we do here:

https://github.com/dotnet/aspire/blob/7378176c2128f5cc26764a48e118212a0a3cbfe5/tests/Aspire.Hosting.Qdrant.Tests/AddQdrantTests.cs#L179-L180